docs/examples/ShaderExample/src/ShaderExample.ls

```diff
@@ -0,0 +1,170 @@
+"varying vec2 v_texcoord0;                                                  \n" +
+"varying vec4 v_color0;                                                     \n" +
+"uniform mat4 u_mvp;                                                        \n" +
+"void main()                                                                \n" +
+"{                                                                          \n" +
+"    gl_Position = u_mvp * a_position;                                      \n" +
+"    v_color0 = a_color0;                                                   \n" +
+"    v_texcoord0 = a_texcoord0;                                             \n" +
+"}";
+
+        private var invertFragmentShader:String =
+"uniform sampler2D u_texture;                                               \n" +
+"varying vec2 v_texcoord0;                                                  \n" +
+"varying vec4 v_color0;                                                     \n" +
+"void main()                                                                \n" +
+"{                                                                          \n" +
+"    vec4 c = texture2D(u_texture, v_texcoord0);                            \n" +
+"    gl_FragColor =  vec4(1.0 - c.r, 1.0 - c.g, 1.0 - c.b, c.a) * v_color0; \n" +
+"}";
+
+        private var grayscaleFragmentShader:String =
+"uniform sampler2D u_texture;                                               \n" +
+"varying vec2 v_texcoord0;                                                  \n" +
+"varying vec4 v_color0;                                                     \n" +
+"void main()                                                                \n" +
+"{                                                                          \n" +
+"    vec4 c = texture2D(u_texture, v_texcoord0);                            \n" +
+"    float gray = c.r + c.g + c.b / 3.0;                                    \n" +
+"    gl_FragColor =  vec4(vec3(gray), c.a) * v_color0;                      \n" +
+"}";
+
+        private var grayscaleShader:Shader;
+        private var invertShader:Shader;
+        private var brightnessShader:Shader;
+
+        private var time:TimeManager;
+
+        override public function run():void
+        {
+            stage.scaleMode = StageScaleMode.LETTERBOX;
+            stage.color = 0x888888;
+
+            // Load the shaders from string literals
+            grayscaleShader = new Shader();
+            grayscaleShader.load(vertexShader, grayscaleFragmentShader);
+            grayscaleShader.onBind += OnGrayscaleShaderBind;
+
+            invertShader = new Shader();
+            invertShader.load(vertexShader, invertFragmentShader);
+            invertShader.onBind += OnInvertShaderBind;
+
+            // Load this one from assets
+            brightnessShader = new Shader();
+            brightnessShader.loadFromAssets("assets/pulse.vert", "assets/pulse.frag");
```

Comment:
No, but it would be possible. Haven't set out to figure out asset code magic yet :)